server/services/team/service.ts

```diff
@@ -396,13 +396,13 @@ export class TeamService extends ServiceClass implements ITeamService {
 			throw new Error('user-not-on-private-team');
 		}
 		if (getAllRooms) {
-			const teamRoomsCursor = this.RoomsModel.findByTeamId(teamId, { skip, limit });
+			const teamRoomsCursor = this.RoomsModel.findByTeamId(teamId, { skip, limit }, query as FilterQuery<IRoom>);
```

Comment:
You don't need to type cast as `FilterQuery` =) typescript can already infer that from the param ;)
```suggestion
			const teamRoomsCursor = this.RoomsModel.findByTeamId(teamId, { skip, limit }, query);
```